doc/nrf/templates/Integration_template_README.rst

```diff
@@ -81,7 +81,7 @@ Applications and samples
 ************************
```

Comment:
For lines 72-78: What would be the technical details that need additional section in a step-by-step procedure? I think it's too late to discuss technicalities when we're providing steps for integration... Shouldn't these be included earlier?

Maybe this should be placed under Solution architecture, if the architecture is open to extensions and/or requires some technical details that are not immediately related to the architecture, but still relevant for the product? Although, on second thought, I think this "Some title" section is simply not needed...